paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -89,16 +88,18 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
             )
         }
 
-        viewModel.transition.observeEvents(this) { transitionTarget ->
+        viewModel.transition.collectInActivity { transitionEvent ->
```

Comment:
see the comment above, passing the lifecycle avoids duplicated code with fragments.